app/components/node-navbar/template.hbs

```diff
@@ -0,0 +1,34 @@
+{{#ember-wormhole to='secondaryNavbar' renderInPlace=renderInPlace}}
+    <nav class='navbar osf-project-navbar' role='navigation'>
+        <div class='container'>
+            <div class='navbar-header'>
+                <button type='button' class='navbar-toggle collapsed' data-toggle='collapse' data-target='.project-nav'>
```

Comment:
Use [`{{bs-collapse}}`](http://www.ember-bootstrap.com/api/classes/Components.Collapse.html) instead of vanilla bootstrap.